client/core/trade.go

```diff
@@ -201,10 +202,12 @@ func (fs *feeStamped) get() uint64 {
 	return fs.rate
 }
 
-// freshRedeemFeeAge is the expiry age for cached redeem fee rates, past which
-// fetchFeeFromOracle should be used to refresh the rate. See
-// cacheRedemptionFeeSuggestion.
-const freshRedeemFeeAge = time.Minute
+const (
+	// freshRedeemFeeAge is the expiry age for cached redeem fee rates, past which
+	// fetchFeeFromOracle should be used to refresh the rate. See
+	// cacheRedemptionFeeSuggestion.
+	freshRedeemFeeAge = time.Minute
```

Comment:
OK but why?  This just looks like needless churn to me.

Reply:
This must be leftover from a bad rebase. Will remove.